repos/system_upgrade/el8toel9/actors/checkblacklistca/actor.py

```diff
@@ -0,0 +1,18 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import checkblacklistca
+from leapp.models import BlackListCA, BlackListError, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class Checkblacklistca(Actor):
```

Comment:
Please use camel case for actor class, like `CheckBlackListCA`

Reply:
@rjrelyea also use please the "blocklist" wording everywhere instead of blacklist, as we are pushed to do the changes everywhere in upcoming months/years. so updated wording will save us a time/work in future. thanks! do not forget to rename also files/dirs properly.

Reply:
Yeah, in some cases correcting old blacklist usage increases requires increasing blacklist usage in some cases. I'll adjust the camel case.